### DISCUSSION TITLE
deleted both partitions.... any hope ?

### DISCUSSION THREAD

**Discussion by larry-more** [Archfx/duo-de#126] [General]
I am embarrassed to say this but I was careless in following flashing instructions and executed all of them (copy/pasted, thinking I could edit them before executing). Now the phone is stuck in the bootloader, recovery gives me "no command" and adb doesn't recognize the device any longer. my computer recognizes something is attached (says "eject surface duo adb sideload"), but I don't know where to go from here if it is possible.....

**Comment by Archfx** [comment]
From the "no command" screen, you can get into fastbootd using the following key combinations: press and hold the power button and press the volume up key, then release the volume up button. 

Once you are on the fastbootd, check your current slot from the following command,
```shell
fastboot getvar current-slot
```
We have to create a partition for the current slot manually since you have deleted the original one.

```shell
fastboot create-logical-partition system_a 3072
```
You should be able to flash the image now using the following command

```shell
fastboot flash system aosp-arm64-ab-gapps-15.0-[[version]].img
fastboot reboot
```

**Reply by billymild** [reply]
This fixed my issue. THANK YOU!!!!!

**Comment by larry-more** [comment]
brilliant and completely satisfying reply !!
I can't thank you enough for your prompt response, which worked immediately. I had no idea what to do and this got me back on track !
 I am appreciative !

**Comment by Slimrock** [comment]
My phone is on loop. After rebooting, and doing the power key + volume up/down, and rebooting via adb or fastboot, the same "FASTBOOT" shows on my phone scree. Please help.

**Reply by larry-more** [reply]
Not sure if you're in the same situation I was, but when my phone showed fastboot, there were no adb devices showing in my computer terminal, and so I thought there was no connection. I did the key combination a couple times but still came to the same spot.  So I then entered the commands op had indicated, and it worked fine.  I notice that he states you get into "fastbootd" with the key combination; I don't recall now, but I believe my phone did say fastbootD...... with a "d" appended, and I'm thinking I was not back in standard adb fastboot on the phone but at another level. Anyway, give it a try.

Bye the way, I am very much liking this new ROM, even though it's taken some getting used to.
